src/DefinitionResolver.php

```diff
@@ -134,7 +135,13 @@ public function createDefinitionFromNode(Node $node, string $fqn = null): Defini
         $def->parameters = [];
         if ($node instanceof Node\FunctionLike) {
             foreach ($node->getParams() as $param) {
-                $def->parameters[] = $this->prettyPrinter->prettyPrint([$param]);
+                if (!$param->getAttribute('parentNode')) {
```

Comment:
@felixfbecker I could not figure out why those instances do not have a `parentNode` attribute - this is a temporary patch. Could you have a look?